### DISCUSSION TITLE
Welcome to cnmf Discussions!

### DISCUSSION THREAD

**Discussion by codyheiser** [codyheiser/cnmf#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by yotamcons** [comment]
Hi there,
I am trying to run the PBMC tutorial and hit some issues (version 2.0.9)
1. i needed to add 
`if __name__=="__main__": main()`
at the end of cnmf.py
2. the arguments names have changed (--total-workers instead of --n-jobs) or being deprecated ( --show-clustering)
3. the "counts" argument for the prepare command is "counts" instead of "-c","--counts"
4. the combine argument for components has a default, thus instead of reading the "name.nmf_params.df.npz" the code tries to run the default parameters

I think the head of the tutorial should specify what version of the code it is built to work with.

**Reply by codyheiser** [reply]
Hi!

I forked this repository from the original [cNMF repo](https://github.com/dylkot/cNMF) that accompanied the [2019 eLife publication](https://elifesciences.org/articles/43803) from Kotliar, et al.  I simply wrapped the entire workflow in a python package that is more seamlessly run from command line (one command rather than 3-4 consecutive scripts).

The tutorials contained in this repo have not been updated and thus do not reference the updated package.  I would suggest cloning the original repository if you'd like to work through the associated tutorials, as I don't anticipate updating the tutorials for my package wrapper in the near future.

Hope this helps,

Cody
